### PR TITLE
Regular mommis on derelict again

### LINKED ISSUE #6848: Allow derelict MoMMIes to name themselves again
Why was this even removed, how is this any different than diona or station built MoMMIes/Borgs.

Really now.

### PR DESCRIPTION
**Pull request by clusterfack**
I don't know precisely the full plans that N3X15 had for derelict mommis to be more Russian-ized, but they were incomplete and we have no way of knowing if they will ever be completed at this point. 

Therefore, unless N3X15 comes back to finish whatever rework he had planned I am changing the type of mommi spawned on the derelict back to the standard type (with namepicking)
Fixes #6848

### CODE REVIEW

**Comment by gbasood** [comment]
Can't think of anything cool to do with MoMMIs on derelict so this is fine :+1: 
Although, maybe we should have a MoMMI spawner on main station if we're spawning normal MoMMIs with normal keeper laws on derelict now

File: code/game/machinery/mommi_spawner.dm [hunk 1]
```diff
@@ -140,7 +140,7 @@
 	if(!user.client) return // Player has already been made into another mob before this one spawned, so don't make a new one
 	var/turf/T = get_turf(src)
 
-	var/mob/living/silicon/robot/mommi/M = new /mob/living/silicon/robot/mommi/soviet(T)
+	var/mob/living/silicon/robot/mommi/M = new /mob/living/silicon/robot/mommiT)
```
Comment (Shadowmech88) [inline code/game/machinery/mommi_spawner.dm]: Shouldn't this be mommi(T) instead of mommiT)?

Comment (clusterfack) [inline code/game/machinery/mommi_spawner.dm]: yes

**Comment by clusterfack** [comment]
We aren't spawning normal mommis with keeper laws

**Comment by gbasood** [comment]
Well alright then

**Comment by Skullyton** [comment]
I remember N3x15 asking spriters to make a more russian MoMMI sprite that would be their default and only sprite, to remove further individuality. The reason being so that you couldn't just have MoMMIs on the derelict dying, clicking the MoMMI spawner, entering the same name, and going "Okay, I'm back" or something along those lines.

**Comment by clusterfack** [comment]
He really wanted to make the derelict an old abandoned russian station where the mommis were the only thing that survived the nuclear blast and with some more LORE shit or something. I thought it was a pretty cool idea but since it was never finished...

**Comment by Skullyton** [comment]
Well, what needed to be done so it was considered finished wasn't exactly outlined beyond needing sprites.

I can probably get some sorted right now, thinking on it

**Comment by TermedClepe** [comment]
>I remember N3x15 asking spriters to make a more russian MoMMI sprite that would be their default and only sprite, to remove further individuality

How horrifying.

I don't see the connection between being russian and having no individuality options. If they could only pick from some list of sprites unique to derelict that would be cool, but limiting name options to predetermined strings is shitty.

**Comment by Skullyton** [comment]
>I don't see the connection between russian and having no individuality

Why would a robot designed for repair need individuality, especially on a station designed more towards efficiency rather than individuality.

I think the main point of separating Derelict MoMMIs from regular station MoMMIS was so that it could be considered a completely new life (So you couldn't have [MoMMI name] V2 when you bite the dust to the point you need to respawn from the MoMMI fabricator, or name yourself something like TERREL IS TRAIT)

Heck, at least Derelict MoMMIs weren't shifted to their own damage control like was originally intended (Usherp or something along those lines, it's just called that for derelict MoMMIs)

**Comment by Shadowmech88** [comment]
>Why would a robot designed for repair need individuality, especially on a station designed more towards efficiency rather than individuality.

It wouldn't.

But would it improve the experience of players who play Derelict MoMMI if we disallowed them from picking names and icons?
Would Derelict MoMMI players have more fun if they weren't allowed to choose such things?

Your stance makes sense from an RP perspective, but the people who play Derelict MoMMI aren't usually concerned with RP, they just want to have a nice comfy round of building, possibly alongside other Derelict MoMMIs. I imagine such players would enjoy the individuality that comes from a unique name, in order to facilitate easy and natural communication between each other.

**Comment by Skullyton** [comment]
It's a bit of a conundrum, you don't want people to use information from their past life in a way to give said information to others (through names), but you also don't want to remove the individuality from them because the actions of the few screw over the many.

Soviet hat machine on the derelict when?

**Comment by Shadowmech88** [comment]
>you don't want people to use information from their past life in a way to give said information to others (through names)

There are a lot more efficient ways for players to do that besides becoming a Derelict MoMMI.
You could become a pAI or diona, which can pick their own names and are capable of talking directly to humans. It doesn't even have to be via namepick, if you wanted to call someone out as traitor you could just as easily inhabit a borer or posibrain and just say it out loud.

Derelict MoMMIs can't even leave their Z-level, and rarely get human visitors. As far as the risk for conveying information through names goes, Derelict MoMMIs are pretty low on the totem pole.

And if a Derelict MoMMI, despite all this, still puts metainformation in their name? Just ban them, like you would if they'd chosen to be a diona or pAI or posibrain or any other method of communicating metainformation.

**Comment by Skullyton** [comment]
Fair enough, but I still want to do a soviet MoMMI sprite for the derelict MoMMIs now

**Comment by TermedClepe** [comment]
Soviet themed sprites are fine, just don't force a single sprite.

**Comment by clusterfack** [comment]
We can set the default starting sprite to the soviet one if you make it @Skullyton, sounds cool

**Comment by AutismSpider** [comment]
make set of soviet/russian icons for derelict mommis and station base mommis have there own

**Comment by SkowronX** [comment]
Someone did make a soviet mommi at some point. It was on one of the older prs.

**Comment by FudgePucker** [comment]
Merge it, eggman left us this mess. There's better ways to reveal round sensitive information than going derelict mommi.

They need communication to the station mommis in case you want to set up an TEG and the admins are ignoring you.

**Comment by clusterfack** [comment]
Eh I'll just put it in, it seems everyone really wants it

**Comment by 9600bauds** [comment]
Democracy wins again

**Pull request merged**